producers/letsCube.py

```diff
@@ -202,7 +212,18 @@ def global_except_hook(exctype, value, exc_traceback):
     if addBinVars is not None and isinstance(addBinVars, dict):
         for cubeName, cube in ana.cubes.items():
             cube.add_BinVar(addBinVars)
+        nBins *= len(addBinVars[1])
             
+    #figure out how many bins & cores we have to maybe add a fake variable.
```

Comment:
Should we try hiding this inside smalldataana_psana, like it was done before? 
I don't feel too strongly about it, just like to keep letsCube.py somewhat compact...

Reply:
I would like to push, but agree that we should create an issue that discusses moving this technical feature 'lower'.